modules/nf-core/gget/gget/main.nf

```diff
@@ -0,0 +1,37 @@
+process GGET_GGET {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::gget=0.27.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gget:0.27.2--pyh4a94de4_2':
+        'quay.io/biocontainers/gget:0.27.2--pyh4a94de4_2' }"
+
+    input:
+    tuple val(meta), path(files)
+
+    output:
+    tuple val(meta), path("*[!versions.yml][!${prefix}.${suffix}]*"), emit: files, optional: true
+    tuple val(meta), path("${prefix}.${suffix}")                   , emit: output, optional: true
+    path "versions.yml", emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def inputs = files ?: ""
+    prefix = task.ext.prefix ?: "${meta.id}"
+    suffix = task.ext.suffix ?: "json"
```

Comment:
Please no `task.et.suffix`

Reply:
Why? I thought we agreed with the maintainers that that one was accepted too?

Reply:
I don't see any htere: https://github.com/nf-core/modules/search?q=task.ext.suffix

Reply:
`gawk`, `bedtools/sort`, `qualimap/bamqc`, `seqkit/replace`... are examples of modules that already use it (damn github search functionality :grin:), I'm always searching in VS code because of it